use/6/b.txt

```diff
@@ -1,15 +1,21 @@
-# USE6 Workflows
+# USE6 Workflow Management Systems
```

Comment:
This should also contain the work overview, similar to SD8

Reply:
done, is pushed.